ansible/roles/sushi-installer/scenarios/repository.yml

```diff
@@ -62,11 +62,6 @@
     - "{{ go_path }}/src/github.com/opensds/nbp/csi/deploy"
     - "{{ go_path }}/src/github.com/opensds/nbp/csi/examples"
 
-- name: copy sushi flexvolume binary file into sushi work directory
```

Comment:
Is it possible to comment the code mentioning this support is not available going forward? We will remove it in next release, if no pressing need comes. (Just moving from "Young generation to Survivor space ;))

Reply:
resolved